polaris-core/src/main/java/org/apache/polaris/core/PolarisConfiguration.java

```diff
@@ -150,4 +150,21 @@ public static <T> Builder<T> builder() {
               "If set to true, allows tables to have external locations outside the default structure.")
           .defaultValue(false)
           .build();
+
+  public static final PolarisConfiguration<Boolean> CLEANUP_ON_NAMESPACE_DROP =
+      PolarisConfiguration.<Boolean>builder()
+          .key("CLEANUP_ON_NAMESPACE_DROP")
+          .catalogConfig("cleanup.on.namespace.drop")
+          .description("If set to true, clean up data when a namespace is dropped")
+          .defaultValue(false)
+          .build();
+
+  public static final PolarisConfiguration<Boolean> DROP_WITH_PURGE_ENABLED =
+      PolarisConfiguration.<Boolean>builder()
+          .key("ALLOW_DROP_WITH_PURGE")
+          .catalogConfig("dropWithPurge.enabled")
```

Comment:
I'm not sure if we have many other examples of camelCase configs yet, but I recall for some other config keys we wanted to stick to `kebab-case` for consistency with other Iceberg conventions. So, `drop-with-purge` would be more in-line